roles/storage/lvm/tasks/install.yml

```diff
@@ -15,11 +15,43 @@
     - ecryptfs-utils 
     - xfsprogs
 
+- name: Storage | Unmount the temporary drive first
+  sudo: true
+  shell: umount /mnt
+  ignore_errors: yes  
+  when: azure
+
+- name: Storage | Wipe them out. All of them. 
+  sudo: true
+  shell: dd if=/dev/zero of=/dev/sdb bs=512 count=1
+  ignore_errors: yes  
+  when: azure
+
 - name: Storage | Remove existing temporary drive (Azure)
   sudo: true
-  shell: sudo parted --script /dev/sdb rm 1 || true
+  shell: parted --script /dev/sdb rm 1
+  ignore_errors: yes  
+  when: azure
+
+- name: Storage | WTF, Unmount the temporary drive first
```

Comment:
Um, is there something in the azure-cloud that "WTF" refers to that has nothing to do with the *usual* meaning?